hydra-node/src/Hydra/Chain/Direct/Wallet.hs

```diff
@@ -192,15 +166,16 @@ newTinyWallet tracer networkId (vk, sk) queryUTxOEtc = do
 -- checking the output's address.
 applyBlock :: Block -> (Address -> Bool) -> Map TxIn TxOut -> Map TxIn TxOut
 applyBlock blk isOurs utxo = case blk of
-  BlockAlonzo (ShelleyBlock (Ledger.Block _ bbody) _) ->
+  BlockBabbage (ShelleyBlock block _) ->
```

Comment:
Same as queries, we probably want to support 2 eras always, the current and next one.